spec.bs

```diff
@@ -2704,20 +2701,16 @@ To <dfn>fetch trusted signals</dfn> given a [=URL=] |url|, an [=origin=] |script
     1. If |dataVersion| is not null:
       1. If |dataVersion| is not an integer, or is less than 0 or more than 2<sup>32</sup>&minus;1,
         set |signals| to failure and return.
-    1. If |isBiddingSignal| is true, then set |formatVersion| to the result of
-      [=header list/getting a structured field value=] given
-      [:X-fledge-bidding-signals-format-version:] and "`item`" from |headers|.
     1. Set |signals| to the result of [=parsing JSON bytes to an Infra value=] |responseBody|.
   1. Wait for |signals| to be set.
   1. If |signals| is a parsing exception, or if |signals| is not an [=ordered map=], return « null,
     null, null ».
-  1. If |formatVersion| is 2:
-    1. If |signals|["`keys`"] does not [=map/exist=], return « null, null ».
-    1. Set |signals| to |signals|["`keys`"].
-    1. If |signals| is not an [=ordered map=], return « null, null ».
-    1. If |signals|["`perInterestGroupData`"] [=map/exists=] and is an [=ordered map=]:
-      1. [=Assert=] |isBiddingSignal| is true.
-      1. Let |perInterestGroupData| be |signals|["`perInterestGroupData`"].
+  1. If |signals|["`keys`"] does not [=map/exist=], return « null, null ».
+  1. Set |signals| to |signals|["`keys`"].

```

Comment:
This is a little awkward, and this entire method has a bunch of bugs (e.g., line 2711 should be before we set signals, a bunch of the returns only return two values instead of three, some places "set |signals| to null and return failure" (signals isn't even being returned, and wasn't passed in by the caller, so setting it to null doesn't do anything), etc.

I think fixing all of that is beyond the scope of this PR.